app/components/course-page/course-stage-step/first-stage-tutorial-card.hbs

```diff
@@ -1,42 +1,144 @@
+          {{else}}
+            <b class="text-teal-600">Step 1:</b>
+          {{/if}}
+          Open a file
+        </div>
+      </div>
+      <CoursePage::CourseStageStep::FirstStageTutorialCard::NavigateToFileStep
+        @repository={{@repository}}
+        @courseStage={{@courseStage}}
+        @isComplete={{this.navigateToFileStepIsComplete}}
+      />
+
+      <div class="prose dark:prose-invert mt-6 mb-3 pt-4 border-t dark:border-white/5">
+        <div class="inline-flex items-center gap-1">
+          {{#if this.hasPassedTests}}
+            {{svg-jar "check-circle" class="w-6 h-6 text-teal-500 inline-flex mb-0.5"}}
+          {{else}}
+            <b class="text-teal-600">Step 2:</b>
+          {{/if}}
+          Uncomment code
+        </div>
+      </div>
+      <div class="max-w-2xl">
         <CoursePage::CourseStageStep::FirstStageTutorialCard::UncommentCodeStep
           @repository={{@repository}}
           @courseStage={{@courseStage}}
           @isComplete={{this.uncommentCodeStepIsComplete}}
         />
-      {{else if (eq stepList.expandedStep.id "submit-code")}}
-        <CoursePage::CourseStageStep::FirstStageTutorialCard::SubmitCodeStep @isComplete={{this.submitCodeStepIsComplete}} />
-      {{/if}}
-    </ExpandableStepList>
-
-    {{! After step 2, step 3 already contains a note on Tests Failed, so we can hide this !}}
-    {{#unless this.uncommentCodeStepIsComplete}}
-      <div class="prose dark:prose-invert prose-sm prose-compact mt-5">
+      </div>
+
+      <div class="prose dark:prose-invert mt-8 mb-3 pt-4 border-t dark:border-white/5">
+        <div class="inline-flex items-center gap-1">
+          {{#if this.hasPassedTests}}
+            {{svg-jar "check-circle" class="w-6 h-6 text-teal-500 inline-flex mb-0.5"}}
+          {{else}}
+            <b class="text-teal-600">Step 3:</b>
+          {{/if}}
+          Submit changes
+        </div>
+      </div>
+      <div class="prose dark:prose-invert mb-3">
         <p>
-          {{svg-jar "information-circle" class="w-5 h-5 mb-1 inline-flex text-sky-500"}}
-          <b>Note:</b>
-          After your first Git push, you should see
-          <code class="font-semibold text-red-700 dark:text-red-300 bg-red-100 dark:bg-red-900/30 border border-red-200 dark:border-red-800/40">Tests
-            failed</code>
-          in the bar below this card. This is expected! Complete the steps above to pass the tests.
+          Save your changes and run the following commands to submit your changes:
+        </p>
+      </div>
+
+      <CopyableTerminalCommand @commands={{array 'git commit -am "[any message]"' "git push origin master"}} class="mb-3" />
+
+      <div class="prose dark:prose-invert">
+        <p class={{if this.hasPassedTests "line-through"}}>
+          Once you run the commands above, the
+          <code class="font-semibold text-red-700 bg-red-100 border border-red-200">Tests failed</code>
+          message below should be changed to
```

Comment:
@andy1li it looks like we're tweaking verbiage too here? This doesn't match the text we're using currently. 

Can you try to minimize the changes here? Small things can make big differences in experiments, ideally we test one aspect at a time